.github/workflows/test-docs-build.yml

```diff
@@ -27,17 +27,15 @@ jobs:
           yarn build
 
       # docs.rs also uses nightly so it makes sense to mirror that
-      - name: Install Rust nightly toolchain
-        uses: actions-rs/toolchain@v1
+      - name: Setup Rust
+        uses: './.github/actions/rust/rust-setup'
         with:
           toolchain: nightly
-          override: true
-
+          os: ${{ runner.os }}
+          job: ${{ github.job }}
+      
       - name: Test Rust Documentation
-        uses: actions-rs/cargo@v1
         env:
           RUSTDOCFLAGS: "-D warnings --cfg docsrs"
-        with:
-          command: doc
-          toolchain: nightly
-          args: --all-features --no-deps --workspace
+          RUSTUP_TOOLCHAIN: "nightly"
+        run: cargo doc --all-features --no-deps --workspace
```

Comment:
Does this definitely run with nightly? Maybe we can use `cargo +nightly doc ...` instead to make sure.

Reply:
According to the docs it [def](https://rust-lang.github.io/rustup/overrides.html) should work, but will change to override shorthand if that is more idiomatic.

Reply:
Alright, then it's fine either way. Thanks!